lib/commands/element.js

```diff
@@ -40,28 +41,57 @@ commands.setValue = function (keys, elementId) {
     unicodeKeyboard: this.opts.unicodeKeyboard
   };
 
-  return this.bootstrap.sendAction("element:setText", p);
+  return await this.bootstrap.sendAction("element:setText", p);
 };
 
-commands.getText = function (elementId) {
-  return this.bootstrap.sendAction("element:getText", {elementId});
+commands.getText = async function (elementId) {
+  return await this.bootstrap.sendAction("element:getText", {elementId});
 };
 
-commands.clear = function (elementId) {
-  return this.bootstrap.sendAction("element:clear", {elementId});
+commands.clear = async function (elementId) {
+  return await this.bootstrap.sendAction("element:clear", {elementId});
 };
 
-commands.click = function (elementId) {
-  return this.bootstrap.sendAction("element:click", {elementId});
+commands.click = async function (elementId) {
+  return await this.bootstrap.sendAction("element:click", {elementId});
 };
 
-commands.getLocation = function (elementId) {
-  return this.bootstrap.sendAction("element:getLocation", {elementId});
+commands.getLocation = async function (elementId) {
+  return await this.bootstrap.sendAction("element:getLocation", {elementId});
 };
 
-commands.getSize = function (elementId) {
-  return this.bootstrap.sendAction("element:getSize", {elementId});
+commands.getSize = async function (elementId) {
+  return await this.bootstrap.sendAction("element:getSize", {elementId});
 };
+
+commands.touchLongClick = async function (elementId, x, y, duration) {
```

Comment:
can we name these "params" instead of "opts" to be more consistent with the other commands?